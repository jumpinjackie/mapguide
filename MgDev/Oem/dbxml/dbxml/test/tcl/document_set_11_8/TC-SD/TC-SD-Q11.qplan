<XQuery>
  <FLWOR>
    <ForBinding name="a">
      <QueryPlanToAST>
        <StepQP axis="child" name="q" nodeType="element">
          <StepQP axis="child" name="qp" nodeType="element">
            <StepQP axis="child" name="s" nodeType="element">
              <StepQP axis="child" name="ss" nodeType="element">
                <NodePredicateFilterQP uri="" name="#tmp2">
                  <StepQP axis="child" name="e" nodeType="element">
                    <StepQP axis="child" name="dictionary" nodeType="element">
                      <ValueQP index="node-element-equality-string" operation="eq" child="hw" value="the"/>
                    </StepQP>
                  </StepQP>
                  <NodePredicateFilterQP uri="" name="#tmp3">
                    <StepQP axis="child" name="hwg" nodeType="element">
                      <VariableQP name="#tmp2"/>
                    </StepQP>
                    <ValueFilterQP comparison="eq" general="true">
                      <StepQP axis="child" name="hw" nodeType="element">
                        <VariableQP name="#tmp3"/>
                      </StepQP>
                      <Sequence>
                        <AnyAtomicTypeConstructor value="the" typeuri="http://www.w3.org/2001/XMLSchema" typename="string"/>
                      </Sequence>
                    </ValueFilterQP>
                  </NodePredicateFilterQP>
                </NodePredicateFilterQP>
              </StepQP>
            </StepQP>
          </StepQP>
        </StepQP>
      </QueryPlanToAST>
    </ForBinding>
    <Sort>
      <Specification modifier="ascending|empty_least">
        <TreatAs>
          <PromoteUntyped uri="http://www.w3.org/2001/XMLSchema" name="string">
            <Atomize>
              <QueryPlanToAST>
                <StepQP axis="child" name="qd" nodeType="element">
                  <VariableQP name=":a"/>
                </StepQP>
              </QueryPlanToAST>
            </Atomize>
          </PromoteUntyped>
          <SequenceType occurrence="zero_or_one" testType="anything"/>
        </TreatAs>
      </Specification>
    </Sort>
    <DOMConstructor type="element">
      <Name>
        <Sequence>
          <ATQNameConstructor uri="" prefix="" localname="Output" typeuri="http://www.w3.org/2001/XMLSchema" typename="QName"/>
        </Sequence>
      </Name>
      <Children>
        <ContentSequence>
          <QueryPlanToAST>
            <StepQP axis="child" name="a" nodeType="element">
              <VariableQP name=":a"/>
            </StepQP>
          </QueryPlanToAST>
        </ContentSequence>
        <ContentSequence>
          <QueryPlanToAST>
            <StepQP axis="child" name="qd" nodeType="element">
              <VariableQP name=":a"/>
            </StepQP>
          </QueryPlanToAST>
        </ContentSequence>
      </Children>
    </DOMConstructor>
  </FLWOR>
</XQuery>
