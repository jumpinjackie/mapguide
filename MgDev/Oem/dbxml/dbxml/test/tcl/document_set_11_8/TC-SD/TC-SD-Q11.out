<Output><a>Mehul Michaelski</a><qd>1058</qd></Output><Output><a>Tsutomu Scharinger</a><qd>1092</qd></Output><Output><qd>1235</qd></Output><Output><a>Kumiko Rueda</a><qd>1276</qd></Output><Output><a>LiWu Alpay</a><qd>1318</qd></Output><Output><a>Wilhelm Munro</a><qd>1372</qd></Output><Output><a>Jingling Alameldin</a><qd>1404</qd></Output><Output><qd>1476</qd></Output><Output><qd>1557</qd></Output><Output><qd>1581</qd></Output><Output><a>Siranush Marchesini</a><qd>1702</qd></Output><Output><a>Ricarda Murer</a><qd>1718</qd></Output><Output><qd>174</qd></Output><Output><a>Colm Vittorini</a><qd>1835</qd></Output><Output><a>Nader Coscoy</a><qd>1922</qd></Output><Output><qd>195</qd></Output><Output><a>Anh Acklen</a><qd>196</qd></Output><Output><a>Ashoke Kahn</a><qd>1999</qd></Output><Output><qd>208</qd></Output><Output><a>Jingling Alameldin</a><qd>219</qd></Output><Output><a>Wilson Sciarra</a><qd>281</qd></Output><Output><a>Xiao Srini</a><qd>331</qd></Output><Output><qd>347</qd></Output><Output><a>Kyuchul Bhavnani</a><qd>379</qd></Output><Output><a>Asiri Shijie</a><qd>411</qd></Output><Output><a>Dagobert Darbha</a><qd>437</qd></Output><Output><qd>458</qd></Output><Output><a>Rongheng Appelfeller</a><qd>509</qd></Output><Output><a>Henri Lecon</a><qd>554</qd></Output><Output><qd>67</qd></Output><Output><a>Garnik Mussi</a><qd>696</qd></Output><Output><a>Trudy Schreckenghost</a><qd>7</qd></Output><Output><qd>77</qd></Output><Output><qd>864</qd></Output><Output><a>Seong Gluck</a><qd>984</qd></Output><Output><a>Randi Tomek</a><qd>985</qd></Output>